compat/test/browser/suspense.test.js

```diff
@@ -586,7 +509,52 @@ describe('suspense', () => {
 		});
 	});
 
-	it('should allow children to update state while suspending', () => {
+	it('should keep hook state of siblings when suspending', () => {
```

Comment:
This test (and the other added test below it) are here just to ensure that hook state and class state have the same behavior in various Suspense scenarios, which wasn't true before. These tests fail on master.